lib/ui/page/home/page/chat/controller.dart

```diff
@@ -1184,6 +1185,77 @@ class ChatController extends GetxController {
     }
   }
 
+  /// Downloads the provided [ImageAttachment] or [FileAttachment].
+  Future<void> downloadMedia(Attachment attachment, {String? to}) async {
+    try {
+      await CacheWorker.instance
+          .download(
+            attachment.original.url,
+            attachment.filename,
+            attachment.original.size,
+            checksum: attachment.original.checksum,
+            to: to,
+          )
+          .future;
+
+      MessagePopup.success(
+        attachment is ImageAttachment
+            ? 'label_image_downloaded'.l10n
+            : 'label_video_downloaded'.l10n,
+      );
+    } catch (_) {
+      MessagePopup.error('err_could_not_download'.l10n);
+    }
+  }
+
+  /// Downloads the provided [ImageAttachment] or [FileAttachment] and saves it
+  /// to the gallery.
```

Comment:
```suggestion
  /// Saves the provided [attachment] to the gallery.
```

Чуть лаконичнее, как Вам кажется?

Reply:
Согласен, так и сделаю.